app/parse.py

```diff
@@ -8,5 +13,26 @@
     duration: str
 
 
+def get_course(course: Tag) -> Course:
+    return Course(
+        name=course.select_one(".ProfessionCard_title__Zq5ZY > h3").text,
+        short_description=(
+            course.select_one(".typography_landingTextMain__Rc8BD.mb-32").text
```

Comment:
Ensure that the CSS selector `.typography_landingTextMain__Rc8BD.mb-32` is correct and matches the structure of the webpage. If the webpage structure changes, this selector might break.